src/classes/UTIL_Describe.cls

```diff
@@ -393,6 +440,48 @@ public class UTIL_Describe {
         }
         return strFields.substring(0, strFields.length()-1);
     }
+	
+    /*******************************************************************************************************
+     * @description Returns a map of active, user-available picklist values and labels for a given
+     * SObjectType and field, keyed by picklist label with a value of picklist value.
+     * @param objectName Name of the object to retrieve picklist values for.
+     * @param fieldName Name of the field to retrieve picklist values for.
+     * False if we want it to be the friendly name.
+     * @return Map<String,String> Map of picklist labels to values for the object and field specified.
+     */
+    @AuraEnabled
+    public static Map<String,String> getPicklistActiveValuesMap(String objectName, String fieldName) {
```

Comment:
This method seems to have some spacing stuff, and if we could add spaced between method arguments, parameter declarations, etc..see lines 454 & 456

Reply:
This has been tweaked to match the standard.